.github/workflows/sanity-test.yml

```diff
@@ -215,6 +219,37 @@ jobs:
       - name: Build ${{ matrix.project }}
         run: python3 builder.pyz build --project ${{ matrix.project }}
 
+  aws-c-io:
+    strategy:
+      fail-fast: false
+      matrix:
+        linux:
+          - manylinux1-x86
+          - manylinux1-x64
+          - manylinux2014-x86
+          - manylinux2014-x64
+          - ubuntu-16-x64
+    
+    needs: package
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Source
+      uses: actions/checkout@v1
+
+    - name: Get release tag
+      uses: ./.github/actions/release-tag
+      id: release
+      with:
+        output: tag
+
+    - name: Build ${{ env.PACKAGE_NAME }}
+      run: |
+        echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u awslabs --password-stdin
+        # TODO: change this to latest, let the version be dictated by --version below
```

Comment:
is this a "todo before merge" or "todo someday"?

Reply:
Before merge